.github/actions/setup-linux/action.yml

```diff
@@ -36,43 +36,43 @@ runs:
   steps:
     - name: Install non-library dependencies
       run: |
-        sudo apt update
-        sudo apt install imagemagick libjpeg-turbo8-dev libpng-dev
+        sudo apt update -y
+        sudo apt install -y imagemagick libjpeg-turbo8-dev libpng-dev
```

Comment:
@vigneshvg @y-guyon We are using brew and vcpkg on Mac and Windows. I wonder if they have a similar option.

I searched for "-y" and "yes" in the [brew man page](https://docs.brew.sh/Manpage) but didn't find anything. Same for the [vcpkg install doc page](https://learn.microsoft.com/en-us/vcpkg/commands/install#options).

Reply:
> @vigneshvg @y-guyon We are using brew and vcpkg on Mac and Windows. I wonder if they have a similar option.
> 
> I searched for "-y" and "yes" in the [brew man page](https://docs.brew.sh/Manpage) but didn't find anything. Same for the [vcpkg install doc page](https://learn.microsoft.com/en-us/vcpkg/commands/install#options).

yeah i was just searching for it as well. it seems like bre and vcpkg assume -y by default and they don't ask for any user prompts.